test/test_jit.py

```diff
@@ -4179,6 +4179,76 @@ def test_list_remove():
             return a == [1, 2, 4]
         self.checkScript(test_list_remove, ())
 
+    def test_mutable_list_index_not_existing(self):
+        @torch.jit.script
+        def test_list_index_not_existing():
+            a = [4, 1, 3, 2]
+            i = a.index(5)
+
+            return i
+
+        with self.assertRaisesRegex(RuntimeError, "'5' is not in list"):
+            test_list_index_not_existing()
+
+    def test_mutable_list_index(self):
```

Comment:
a very **minor** nitpick: i wonder if `mutable` qualifier adds any value here? Aren't all `list`s mutable in python, we could just use `int_list`, `tensor_list` unless you are referring to an underlying element type, then `int`s should be immutable?

Reply:
@Krovatkin I changed the name of the tests.

Reply:
thank you 😄 !

Reply:
Not that it needs changing again, but on the other hand the existing convention  these new tests were following the convention already set in test_jit.py, like with test_mutable_list_append. It also makes easier to grep for mutable changes. (again not that it needs to be rechanged).